pydoctor/model.py

```diff
@@ -789,7 +789,7 @@ def analyzeModule(self,
         self.progress(
             "analyzeModule", len(self.allobjects),
             None, "modules and packages discovered")
-        self.unprocessed_modules.add(mod)
+        self.unprocessed_modules.append(mod)
```

Comment:
Now that we have a list, should this check for duplicate modules ?

At the same time, the de-duplicate can be done later when we start processing the unprocessed modules.

But I prefer to check for duplication at append time.

But maybe the way `analyzeModule` works could not lead to duplicate modules.